### ISSUE TITLE
Smart Double Tap/Double Tap Ability to Self Cast not working in specific situations

### ISSUE THREAD

**Issue by ElectricSoap1** [ValveSoftware/Dota2-Gameplay#9281]
### Description

Certain items and abilities aren't currently working in Dota 2 when using Smart Double Tap or Double Tap Ability to Self Cast. Examples include but are not limited to Mjollnir and Necro's Shard, you can use those items and abilities respectfully if self casting using the cursor but not when Alt+button press or double tapping button press. The correct settings are selected as other things work properly like Bloodseekers Q for example.

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/111933407/e9418daa-d8cd-49e8-a09e-68ab23e5f36c)

**Comment by VvaRGoD2007** [comment]
is this fixed already?

**Comment by VigneshMohan1698** [comment]
Yeah it's not working on simple items like hurricane pike. It's really annoying.
